.github/workflows/sanity-check.yaml

```diff
@@ -0,0 +1,10 @@
+name: "Test"
+on: [pull_request]
+
+jobs:
+  check-container-scanning-examples:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Ensure example docker images can be built
+        run: make build-images
```

Comment:
Let's make sure this works with docker buildkit while we're in CI.  `DOCKER_BUILDKIT=1` works in my local shell.